src/test/java/seedu/address/logic/commands/ListEntryCommandTest.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.ALL_PREDICATE;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalTeachingAssistant.getTypicalTeachingAssistant;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+
+public class ListEntryCommandTest {
+
+    private Model model;
+    private Model expectedModel;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalTeachingAssistant(), new UserPrefs());
+        expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+
+    }
+
+    @Test
+    public void execute_listIsNotFiltered_showsSameList() {
```

Comment:
i expected more tests that involve not just this predicate for better coverage but i guess it's fine

Reply:
Ideally this should include testing for day/week, but its hard to implement it consistently since the date time is constantly changing. A valid test for day/week might be valid today will become invalid few days later causing problems